### PR TITLE
Added remotes constraints to nova service

### PR DESCRIPTION
**Pull request by tboerger**
**Depends on https://github.com/crowbar/crowbar-core/pull/193**

### CODE REVIEW

File: crowbar_framework/app/models/nova_service.rb [hunk 1]
```diff
@@ -44,14 +44,16 @@ def role_constraints
           "exclude_platform" => {
             "suse" => "< 12.1",
             "windows" => "/.*/"
-          }
+          },
+          "remotes" => true
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: Not for docker :-)

File: crowbar_framework/app/models/nova_service.rb [hunk 2]
```diff
@@ -44,14 +44,16 @@ def role_constraints
...
         },
         "nova-compute-hyperv" => {
           "unique" => false,
           "count" => -1,
           "platform" => {
             "windows" => "/.*/"
-          }
+          },
+          "remotes" => true
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: Neither for hyperv.

File: crowbar_framework/app/models/nova_service.rb [hunk 3]
```diff
@@ -75,7 +79,8 @@ def role_constraints
           "exclude_platform" => {
             "suse" => "< 12.1",
             "windows" => "/.*/"
-          }
+          },
+          "remotes" => true
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: And not for vmware.

File: crowbar_framework/app/models/nova_service.rb [hunk 4]
```diff
@@ -83,14 +88,16 @@ def role_constraints
...
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: Not for zvm either.

File: crowbar_framework/app/models/nova_service.rb (hunk 1 above)
Comment (tboerger) [inline crowbar_framework/app/models/nova_service.rb]: Done

File: crowbar_framework/app/models/nova_service.rb (hunk 3 above)
Comment (tboerger) [inline crowbar_framework/app/models/nova_service.rb]: Done

File: crowbar_framework/app/models/nova_service.rb (hunk 4 above)
Comment (tboerger) [inline crowbar_framework/app/models/nova_service.rb]: Done

**Comment by tboerger** [comment]
Github displays the above comments wrong... I have removed the `remotes` flag from zvm, vmware, hyperv and docker.

**Comment by rsalevsky** [comment]
+1

**Comment by vuntz** [comment]
+1, but do not merge yet (would like to review the related PR first) (also, I wonder if `remotes` is the best name for this, but that can easily be fixed later on).

**Comment by vuntz** [comment]
I've included this in https://github.com/crowbar/crowbar-openstack/pull/211 which contains all the backend logic. Thanks @tboerger !

**Pull request closed without merging**